html/components/SimpleHtml.js

```diff
@@ -233,10 +233,13 @@ class SimpleHtml extends PureComponent {
       ol: this.renderOrderedListPrefix,
     };
 
-    const cssRules = cssRulesFromSpecs({
-      ...defaultTableStylesSpecs,
-      ...style.table,
-    });
+    const tableStyle = _.get(style, 'table', {});
+    const tableCssStyle = _.get(style, 'tableCss', '');
+    const cssRules =
+      cssRulesFromSpecs({
+        ...defaultTableStylesSpecs,
+        ...tableStyle,
+      }) + tableCssStyle;
```

Comment:
Maybe like this?

```
const stylesCssString = cssRulesFromSpecs({
        ...defaultTableStylesSpecs,
        ...tableStyle,
      });
   const cssRules = `${stylesCssString}${tableCssStyle}`;
```

Seems easier to read, but up to you.
You've already got approvals, merge when you want

Reply:
Good point, agree that it's easier to read.
I renamed `stylesCssString` to just `cssStyle` so it's shorter and similar name to `tableCssStyle`, hope that's alright.